tests/InMemoryWallet.ts

```diff
@@ -13,6 +13,7 @@ import BigNumber from 'bn.js'
 
 import { convertToAskarKeyBackend } from '../packages/askar/src/utils/askarKeyBackend'
 import { didcommV1Pack, didcommV1Unpack } from '../packages/askar/src/wallet/didcommV1'
+import { expandIfPossible } from '../packages/core/src/crypto/jose/jwk/ecCompression'
```

Comment:
Should this method be available as export? Or on a key instance or something?


E.g. a getter with compressedPubliKey that optionally returns null?